src/entity/effect/VanillaEffects.php

```diff
@@ -58,6 +58,7 @@
  * @method static Effect WATER_BREATHING()
  * @method static Effect WEAKNESS()
  * @method static WitherEffect WITHER()
+ * @method static SlowFallingEffect Slow_Falling()
```

Comment:
You shouldn't manually write this, it's autogenerated

Reply:
How does it

Reply:
How does it work?

Reply:
https://github.com/pmmp/PocketMine-MP/blob/master/build/generate-registry-annotations.php

Reply:
it's good I just did with generate-registry-annotations for the annatotion of the sloaw falling in the vanillaEffect and the change of the names after the generation of the method